ExportAnnotations command line only works on Windows

## Description

https://github.com/tenacom/Cecil.XmlDocNames/pull/14/checks?check_run_id=1234814396

```
  Running ExportAnnotations...
  Could not execute because the specified command or file was not found.
  Possible reasons for this include:
    * You misspelled a built-in dotnet command.
    * You intended to execute a .NET Core program, but dotnet-/home/runner/.nuget/packages/resharper.exportannotations.task/1.3.0/tools/netcoreapp3.1ExportAnnotations.dll does not exist.
    * You intended to run a global tool, but a dotnet-prefixed executable with this name could not be found on the PATH.
/home/runner/.nuget/packages/resharper.exportannotations.task/1.3.0/build/ReSharper.ExportAnnotations.Task.Building.targets(76,5): error MSB3073: The command "dotnet /home/runner/.nuget/packages/resharper.exportannotations.task/1.3.0/tools/netcoreapp3.1\ExportAnnotations.dll "/home/runner/work/Cecil.XmlDocNames/Cecil.XmlDocNames/src/Cecil.XmlDocNames/obj/Release/net46/Cecil.XmlDocNames.dll" --liblist "obj/Release/net46/ExportAnnotations.libraries"" exited with code 1. [/home/runner/work/Cecil.XmlDocNames/Cecil.XmlDocNames/src/Cecil.XmlDocNames/Cecil.XmlDocNames.csproj]
```

Notice that the tool path ends in `tools/netcoreapp3.1\ExportAnnotations.dll` (backslash instead of slash).

https://github.com/tenacom/Cecil.XmlDocNames/pull/14/checks?check_run_id=1234814410

```
  Running ExportAnnotations...
  Could not execute because the specified command or file was not found.
  Possible reasons for this include:
    * You misspelled a built-in dotnet command.
    * You intended to execute a .NET program, but dotnet-/home/runner/.nuget/packages/resharper.exportannotations.task/1.3.0/tools/net5.0ExportAnnotations.dll does not exist.
    * You intended to run a global tool, but a dotnet-prefixed executable with this name could not be found on the PATH.
/home/runner/.nuget/packages/resharper.exportannotations.task/1.3.0/build/ReSharper.ExportAnnotations.Task.Building.targets(76,5): error MSB3073: The command "dotnet /home/runner/.nuget/packages/resharper.exportannotations.task/1.3.0/tools/net5.0\ExportAnnotations.dll "/home/runner/work/Cecil.XmlDocNames/Cecil.XmlDocNames/src/Cecil.XmlDocNames/obj/Release/net46/Cecil.XmlDocNames.dll" --liblist "obj/Release/net46/ExportAnnotations.libraries"" exited with code 1. [/home/runner/work/Cecil.XmlDocNames/Cecil.XmlDocNames/src/Cecil.XmlDocNames/Cecil.XmlDocNames.csproj]
```

Notice that the tool path ends in `tools/net5.0\ExportAnnotations.dll` (backslash instead of slash).

## Configuration

Judging from [the source code](https://github.com/tenacom/ReSharper.ExportAnnotations/blob/9fcd8b7e47a8813b826c9f5eb1204be3491216e9/src/ReSharper.ExportAnnotations.Task/build/ReSharper.ExportAnnotations.Task.Building.targets#L17-L31), this always happens when building on a non-Windows system.

## Regression?

There's always been a backslash there (see [here](https://github.com/tenacom/ReSharper.ExportAnnotations/blob/a4f0170e74f430bb2860132251eeaca0611fb9ac/src/ReSharper.ExportAnnotations.Task/build/ReSharper.ExportAnnotations.Task.Building.targets#L22-L24) and [here](https://github.com/tenacom/ReSharper.ExportAnnotations/blob/ecc0567b334e27d3c65ebb96f03bc462db8dd8c1/src/ReSharper.ExportAnnotations.Task/buildCrossTargeting/ReSharper.ExportAnnotations.Task.targets#L16-L18)), so this is not a regression.

## Other information

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Bugs fixed in this release
 
+- https://github.com/tenacom/ReSharper.ExportAnnotations/issues/23 - Builds fail on non-Windows systems.
+
 ### Known problems introduced by this release
 
 ## [1.3.0](https://github.com/tenacom/ReSharper.ExportAnnotations/releases/tag/1.3.0) (2020-10-06)

--- a/src/ReSharper.ExportAnnotations.Task/build/ReSharper.ExportAnnotations.Task.Building.targets
+++ b/src/ReSharper.ExportAnnotations.Task/build/ReSharper.ExportAnnotations.Task.Building.targets
@@ -17,17 +17,17 @@
   <PropertyGroup Condition="'$(MSBuildRuntimeType)' == 'Full'">
     <_ExportAnnotationsTfm Condition="'$(MSBuildVersion)' &lt; '16.0'">net46</_ExportAnnotationsTfm>
     <_ExportAnnotationsTfm Condition="'$(MSBuildVersion)' &gt;= '16.0'">net472</_ExportAnnotationsTfm>
-    <_ExportAnnotationsCommand>$(_ExportAnnotationsToolsDirectory)$(_ExportAnnotationsTfm)\ExportAnnotations.exe</_ExportAnnotationsCommand>
+    <_ExportAnnotationsCommand>$(_ExportAnnotationsToolsDirectory)$(_ExportAnnotationsTfm)/ExportAnnotations.exe</_ExportAnnotationsCommand>
   </PropertyGroup>
   <PropertyGroup Condition="'$(MSBuildRuntimeType)' == 'Core'">
     <_ExportAnnotationsTfm Condition="'$(MSBuildVersion)' &lt; '16.7'">netcoreapp2.1</_ExportAnnotationsTfm>
     <_ExportAnnotationsTfm Condition="'$(MSBuildVersion)' &gt;= '16.7' And '$(MSBuildVersion)' &lt; '16.8'">netcoreapp3.1</_ExportAnnotationsTfm>
     <_ExportAnnotationsTfm Condition="'$(MSBuildVersion)' &gt;= '16.8'">net5.0</_ExportAnnotationsTfm>
-    <_ExportAnnotationsCommand>dotnet $(_ExportAnnotationsToolsDirectory)$(_ExportAnnotationsTfm)\ExportAnnotations.dll</_ExportAnnotationsCommand>
+    <_ExportAnnotationsCommand>dotnet $(_ExportAnnotationsToolsDirectory)$(_ExportAnnotationsTfm)/ExportAnnotations.dll</_ExportAnnotationsCommand>
   </PropertyGroup>
   <PropertyGroup Condition="'$(MSBuildRuntimeType)' == 'Mono'">
     <_ExportAnnotationsTfm>net472</_ExportAnnotationsTfm>
-    <_ExportAnnotationsCommand>mono $(_ExportAnnotationsToolsDirectory)$(_ExportAnnotationsTfm)\ExportAnnotations.exe</_ExportAnnotationsCommand>
+    <_ExportAnnotationsCommand>mono $(_ExportAnnotationsToolsDirectory)$(_ExportAnnotationsTfm)/ExportAnnotations.exe</_ExportAnnotationsCommand>
   </PropertyGroup>
 
   <Target Name="_ExportAnnotations_CheckPreconditions">

